Tools/CISupport/ews-build/steps.py

```diff
@@ -5015,9 +5021,11 @@ def run(self, BufferLogObserverClass=logobserver.BufferLogObserver):
             return rc
 
         log_text = self.log_observer.getStdout().rstrip()
-        if log_text:
-            self.summary = log_text
+        if any(['ChangeLog' in file for file in self._files()]):
+            self.summary = 'ChangeLog modified, WebKit only allows commit messages'
```

Comment:
Might reword it slightly. Maybe: 'ChangeLog modification found, this is not allowed anymore, please use commit message'